web/base_pdf_page_view.js

```diff
@@ -152,6 +193,10 @@ class BasePDFPageView {
     canvas.remove();
     canvas.width = canvas.height = 0;
     this.canvas = null;
+    if (this.#tempCanvas) {
+      this.#tempCanvas.width = this.#tempCanvas.height = 0;
+      this.#tempCanvas = null;
+    }
```

Comment:
Should we perhaps add a new helper method, e.g. `#resetTempCanvas`, to avoid repeating this code verbatim?